package-lock.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Please remove this package-lock.json  as we dont need this here.

Reply:
ready!, thanks